### PR TITLE
Bump `atoms-rendering` to v23.0.2

### PR DESCRIPTION
**Pull request by arelra**
## Why?

Bump `atoms-rendering` to the latest version to bring in:

### v22.6.3
Fix sticky video content layout jump:
https://github.com/guardian/atoms-rendering/pull/374
Unstick YoutubeAtom on pause:
https://github.com/guardian/atoms-rendering/pull/376

### v23.0.0
YoutubeAtom - Fix Duplicate Videos:
https://github.com/guardian/atoms-rendering/pull/365

| Before      | After |
| ----------- | ----------- |
|   Screenshot 2022-03-24 at 11 24 08 | Screenshot 2022-03-24 at 11 24 55 |

### v23.0.1
YoutubeAtom - stop other playing videos when a video plays
https://github.com/guardian/atoms-rendering/pull/373

https://user-images.githubusercontent.com/15894063/161256708-d7620ea1-7d15-457a-af80-625be820df49.mov

### v23.0.2
YoutubeAtom - fix dispatch play event when duplicate videos exist in page
https://github.com/guardian/atoms-rendering/pull/383

https://user-images.githubusercontent.com/15894063/161565631-a053d22c-f89b-4a1f-aa53-c3a6c4cb4ac9.mov

### CODE REVIEW

**Review by marjisound — APPROVED** [review]
LGTM

**Comment by marjisound** [comment]
Only if you think it's worth, I was just wondering if you'd like to update one of the existing cypress tests or add a new one to factor having 2 videos with the same video id?

**Comment by joecowton1** [comment]
Is it worth including the release @marjisound did on Friday in this bump? https://github.com/guardian/atoms-rendering/pull/380

File: dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx [hunk 1]
```diff
@@ -229,7 +232,7 @@ export const YoutubeBlockComponent = ({
 				eventEmitters={[ophanTracking, gaTracking]}
 				pillar={format.theme}
 				origin={process.env.NODE_ENV === 'development' ? '' : origin}
-				shouldStick={stickyVideos}
+				shouldStick={!!stickyVideos}
```
Comment (joecowton1) [inline dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx:235]: I _think_ this is a boolean anyway, it's just that the type definition on line 35 is optional when it doesn't need to be.

**Comment by arelra** [comment]
> Only if you think it's worth, I was just wondering if you'd like to update one of the existing cypress tests or add a new one to factor having 2 videos with the same video id?

Yes it's definitely worth add a test for the duplicate video scenario.

Afaics this doesn't happen on articles so is there a deterministic live/dead blog request we can use to test for this?

**Comment by arelra** [comment]
> Is it worth including the Atoms-Rendering release @marjisound did on Friday in this bump? [guardian/atoms-rendering#380](https://github.com/guardian/atoms-rendering/pull/380)

Yes sure. @marjisound if you wish feel free to bump this further with your change 👍

File: dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx (hunk 1 above)
Comment (arelra) [reply dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx]: Good shout thanks. Updated

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 2]
```diff
@@ -174,6 +174,58 @@ describe('YouTube Atom', function () {
...
+		);
+		cmpIframe().contains("It's your choice");
+		cmpIframe().find("[title='Manage my cookies']").click();
+		privacySettingsIframe().contains('Privacy settings');
+		privacySettingsIframe()
+			.find("[title='Accept all']", { timeout: 12000 })
+			.click();
+
+		// Wait for hydration
+		cy.get('[data-component=youtube-atom]')
```
Comment (arelra) [inline dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:190]: Do we need to do this for each atom?

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 3]
```diff
@@ -174,6 +174,58 @@ describe('YouTube Atom', function () {
...
+			.parent()
+			.should('have.attr', 'data-gu-ready', 'true');
+
+		// Make sure overlay for both videos is displayed
+		const overlaySelectorforMultipleVideos = `[data-cy^="youtube-overlay-qkC9z-dSAOE"]`;
+		cy.get(overlaySelectorforMultipleVideos)
+			.should('have.length', 2)
+			.each((item) => {
+				cy.wrap(item).should('be.visible');
```
Comment (arelra) [inline dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:199]: Nice!

**Comment by mxdvl** [comment]
@marjisound, you may want to check your git config, as you're not committing with your GitHub identity!

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 4]
```diff
@@ -174,6 +174,58 @@ describe('YouTube Atom', function () {
...
+		interceptYouTubeEmbed({
+			videoId: 'qkC9z-dSAOE',
+			adUnit: '/59666047/theguardian.com/world/liveblog/ng',
+			pageUrl:
+				'/world/live/2022/mar/28/russia-ukraine-war-latest-news-zelenskiy-putin-live-updates',
+			rejectAll: false,
+		}).as('youtubeEmbed');
+
+		// Play videos
+		cy.get(overlaySelectorforMultipleVideos).each((item) => {
```
[the remark this answers is not in the document]
Comment (marjisound) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js]: Thanks @arelra for pointing this out. I made some changes in the test and tried to follow the flow as you explained in your comment. Does it look any better now? :)

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js (hunk 2 above)
Comment (marjisound) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:190]: I wasn't sure, I just brought it because it was there in the other tests. Do you think this check is not needed?

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 5]
```diff
@@ -174,6 +174,58 @@ describe('YouTube Atom', function () {
 		cy.get(overlaySelector).should('not.exist');
 	});
 
+	it('plays in video when 2 same videos exist in page', function () {
+		// cy.intercept('**youtube**').as('callToYouTube');
+		cy.visit(
+			'/Article?url=https://www.theguardian.com/world/live/2022/mar/28/russia-ukraine-war-latest-news-zelenskiy-putin-live-updates',
```
[the remark this answers is not in the document]
Comment (marjisound) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:180]: Good idea, just changed it 👍

**Comment by marjisound** [comment]
> @marjisound, you may want to check your git config, as you're not committing with your GitHub identity!

Good spot, I don;'t know how my account details have been deleted from my git config. Is `name` & `email` sufficient? Or do you think I can add anything else as well?

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (arelra) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js]: Looks great! I just added a bit more stricter checking with the `times` option to our intercept functions.

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js (hunk 2 above)
Comment (arelra) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:196]: Oh I meant waiting to hydrating each atom. I followed your example below to add this 👍

Comment (marjisound) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:196]: Nice 👍

**Comment by arelra** [comment]
> Great work @arelra @marjisound 👍 I'm a little confused as to what's been approved at this stage - are we all happy with how it's looking?

LGTM. I think it just needs a sanity check on CODE then its good to go.

**Comment by arelra** [comment]
> This was a fantastic work, thanks @arelra and @joecowton1 for all your hard work. I'm shipping it now 🚀

Great work @marjisound and @joecowton1 !
